lib/modelHelpers.py

```diff
@@ -32,40 +32,20 @@ def new_thread():
     actions.app.notify("Created a new thread")
 
 
-def push_context(context: str):
+def push_context(context: dict[str, any]):
     """Add the selected text to the stored context"""
     global stored_context
-    stored_context += [format_message(context)]
+    stored_context += [context]
     actions.app.notify("Appended user context")
 
 
-def push_thread(context: str):
+def push_thread(context: dict[str, any]):
```

Comment:
Once again not relevant to this PR but we will eventually want to have this have the proper roles. i.e. `assistant` for the model responses. That way our model is truly seeing the model history with who did what and not just a series of user strings.